src/wasm-binary.h

```diff
@@ -116,6 +116,7 @@ struct LEB {
   }
 };
 
+#define SIZE32LEB(v) U32LEB((uint32_t)(v))
```

Comment:
Since `SIZE32LEB` performs a useful task, properly casting `size_t` to 32-bit, we should do the opposite. Accept it for now. Perhaps someone can come up with a better solution for `SIZE32LEB` specifically or all of the `*32LEB` cast in general.